uint8 pin_id
bool value
---
bool value
